### DISCUSSION TITLE
HTML vs. XHTML (or, HTML–HTML vs. XHTML–HTML)

### DISCUSSION THREAD

**Discussion by j9t** [Devographics/surveys#160] [State of HTML Design]
Related: [_Write HTML, the HTML Way (Not the XHTML Way)_](https://meiert.com/en/blog/write-html/) (disclosure: author’s reference)

Suggestion to ask about HTML and XHTML use:

* Is XHTML still being used by anyone?
* If yes, with what DOCTYPE?
* Also if yes, with what MIME type?
* For HTML, how is HTML being used—i.e., is it XHTML-inspired (“XHTML–HTML,” closing all elements &c.), or HTML-inspired (like omitting optional markup)?

**Comment by AnthonyLenglet** [comment]
based on self closing tags still being widely used, I feel like you could argue that xhtml is still being used by a lot of people 😄

https://jakearchibald.com/2023/against-self-closing-tags-in-html/
